doc/DOCUMENTATION.org

```diff
@@ -310,12 +313,13 @@ Where:
 | packages.el    | The list of packages to install and the functions to initialize them |
 
 =Packages= are =ELPA= packages which can be installed from an =ELPA= compliant
-repository, and =Extensions= are generally elisp code from git submodules.
+repository, and =Extensions= are generally elisp code from git submodules, or
+other code one might want to use but which isn't available on an =ELPA=
+compliant repository, for whatever reason.
```

Comment:
Or scratch that, I or another contributor will open a new PR to update all this stuff in DOCUMENTATION.org